v2/thirdpartyemailpassword/quick-setup/backend.mdx

```diff
@@ -54,7 +54,10 @@ supertokens.init({
         // learn more about this on https://supertokens.io/docs/thirdpartyemailpassword/appinfo
         appName: "YOUR APP NAME",
         apiDomain: "YOUR API DOMAIN",
-        websiteDomain: "YOUR WEBSITE DOMAIN"
+        websiteDomain: "YOUR WEBSITE DOMAIN",
+        // The default values for apiBasePath and websiteBasePath are /auth
+        apiBasePath: "YOUR API BASE PATH",
+        websiteBasePath: "YOUR WEBSITE BASE PATH"
```

Comment:
- Remove the `apiBasePath` and `websiteBasePath`, and instead, add only about `apiBasePath` in the important tool tip above this section with a short example.